Pages/Index.cshtml.cs

```diff
@@ -1,47 +1,71 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.IO;
+using System.Net;
+
 
 namespace DotnetBunnyLogBrowser.Pages
 {
-	public class IndexModel : PageModel
+    public class IndexModel : PageModel
 	{
-		public string GetExampleLog() => "createdump-aspnet log:\n" + //TODO: delete example log
-			       "\n" +
-			       "+ version=2.1.504\n" +
-			       "+ dotnet new web --force\n" +
-			       "The template \"ASP.NET Core Empty\" was created successfully.\n" +
-			       "\n" +
-			       "Processing post-creation actions...\n" +
-			       "Running 'dotnet restore' on /mnt/ramdisk/dotnet-bunny/dotnet-regular-tests/createdump-aspnet/createdump-aspnet.csproj...\n" +
-			       "Restoring packages for /mnt/ramdisk/dotnet-bunny/dotnet-regular-tests/createdump-aspnet/createdump-aspnet.csproj...\n" +
-			       "Generating MSBuild file /mnt/ramdisk/dotnet-bunny/dotnet-regular-tests/createdump-aspnet/obj/createdump-aspnet.csproj.nuget.g.props.\n" +
-			       "    Generating MSBuild file /mnt/ramdisk/dotnet-bunny/dotnet-regular-tests/createdump-aspnet/obj/createdump-aspnet.csproj.nuget.g.targets.\n" +
-			       "    Restore completed in 1.22 sec for /mnt/ramdisk/dotnet-bunny/dotnet-regular-tests/createdump-aspnet/createdump-aspnet.csproj.\n" +
-			       "\n" +
-			       "Restore succeeded.\n" +
-			       "\n" +
-			       "+ dotnet tool install -g dotnet-dev-certs\n" +
-			       "Tool 'dotnet-dev-certs' is already installed.\n" +
-			       "+ true\n" +
-			       "+ dotnet dev-certs https\n" +
-			       "Cannot find command 'dotnet dev-certs', please run the following command to install\n" +
-			       "\n" +
-			       "dotnet tool install --global dotnet-dev-certs";
-
 		public List<BunnyJob> GetJobs()
 		{
-			//TODO...
+            var job_names=Directory.GetDirectories(JobsConfig.Get().JobsDirectory, JobsConfig.Get().JobsPrefix+"*");

```

Comment:
As much as I prefer variable names with underscores over camelcase, that is not the way the world has gone.